### PR TITLE
Lg/0.5.6 release notes

### PR DESCRIPTION
**Pull request by lsgunnlsgunn**
**Overview**
Release notes for 0.5.6 > 0.5.7

**Requirements**
N/A

**Considered Solutions**
N/A

**Recommended Solution**
N/A

**Considerations**
N/A

### CODE REVIEW

File: modules/developers-guide/pages/cli-reference.adoc [hunk 1]
```diff
@@ -1292,7 +1292,7 @@ If a newer version of `+dfx+` is available, the command automatically downloads
 
 [source,bash]
 ----
-Current version: 0.5.5
+Current version: 0.5.6
```
Comment (hansl) [inline modules/developers-guide/pages/cli-reference.adoc:1295]: 0.5.7?

File: modules/release-notes/pages/sdk-release-notes.adoc [hunk 2]
```diff
@@ -35,14 +35,45 @@ This {release} version of the software and programming language should not be us
 
 The {release} release includes the following new features and capabilities:
 
-- New logging capability that supports logging informational, warning, error, debug, and trace messages to a file in addition to standard error and standard output.
-- Specifying the `--verbose` option generates logs that include debug and trace messages.
+- {proglang}, the Candid interface description language, and the Candid UI now support floating point numbers (float64).
```
Comment (chenyan-dfinity) [inline modules/release-notes/pages/sdk-release-notes.adoc:38]:
```suggestion
- {proglang}, and the JavaScript library now support floating point numbers (Float).
```

File: modules/release-notes/pages/sdk-release-notes.adoc [hunk 3]
```diff
@@ -35,14 +35,45 @@ This {release} version of the software and programming language should not be us
...
+- When using the Candid UI, you can view return values in the following formats:
++
+    - using the interface description language syntax
+    - as values displayed in fields using a browser
+    - as JSON fields.
```
Comment (chenyan-dfinity) [inline modules/release-notes/pages/sdk-release-notes.adoc:43]:
```suggestion
- When using the Candid UI, you can view return values in text, UI widget and JSON formats.
```

File: modules/release-notes/pages/sdk-release-notes.adoc [hunk 4]
```diff
@@ -35,14 +35,45 @@ This {release} version of the software and programming language should not be us
...
++
+You can switch between the formats for representing values by clicking the results bar.
 - Internal updates that are not yet user-visible to support future features.
 
+== Breaking change
+The {release} release includes the following changes that might require updates to existing programs:
+
+- The name of the library recognized by the {proglang} compiler has changed from `stdlib` to `base`.
```
Comment (chenyan-dfinity) [inline modules/release-notes/pages/sdk-release-notes.adoc:51]:
```suggestion
- The name of the built-in {proglang} library has changed from `stdlib` to `base`.
- All of the module names in the `base` library starts with an uppercase letter.
- module `principalId` changed to `Principal`.
```

File: modules/release-notes/pages/sdk-release-notes.adoc [hunk 5]
```diff
@@ -35,14 +35,45 @@ This {release} version of the software and programming language should not be us
...
++
+If you have programs that rely on modules `base` library, you must change the `import` statement to use the new library name.
```
Comment (chenyan-dfinity) [inline modules/release-notes/pages/sdk-release-notes.adoc:53]:
```suggestion
If you have programs that rely on modules `stdlib`, you must change the `import` statement to use the new library name `base`.
```

File: modules/release-notes/pages/sdk-release-notes.adoc [hunk 6]
```diff
@@ -35,14 +35,45 @@ This {release} version of the software and programming language should not be us
...
+For example, you might have previously imported the Array module using the following statement:
++
+[source,motoko]
+----
+import Array "mo:stdlib/Array";
```
Comment (chenyan-dfinity) [inline modules/release-notes/pages/sdk-release-notes.adoc:58]:
```suggestion
import Array "mo:stdlib/array";
```

File: modules/release-notes/pages/sdk-release-notes.adoc [hunk 7]
```diff
@@ -35,14 +35,45 @@ This {release} version of the software and programming language should not be us
...
++
+[source,motoko]
+----
+import Array "mo:base/Array";
+----
+- The modules in the Motoko base library are case-sensitive.
+With this release, all of the module names have been modified to start with an uppercase letter.
+If you have programs that refer to modules in the `base` library using lowercase names, you must change the program to use the new naming convention.
++
+For example, if you previously referred to the `array` module, you would change the program to ref to the `Array` modules.
```
Comment (chenyan-dfinity) [inline modules/release-notes/pages/sdk-release-notes.adoc:71]:
```suggestion
Special note for Mac users: The Mac file system is case-insensitive by default, so you might be able to build your {proglang} program using a lower case module name, but the same code will not build in Linux. Please always capitalize the module names, e.g. `import Array "mo:base/Array"`, not `import Array "mo:base/array"`.
```

**Pull request merged**